<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MoreSetting</class>
 <widget class="QWidget" name="MoreSetting">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>425</width>
    <height>277</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>When desktop is hidden</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="autoBox">
       <item>
        <property name="text">
         <string>continue playing</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>pause</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Video FPS</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="fpsbox">
       <item>
        <property name="text">
         <string>default</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>10</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>15</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>20</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>24</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>25</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>30</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>60</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </item>
   <item row="2" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Decoder</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="hwdecBox">
       <item>
        <property name="text">
         <string>auto</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>gpu</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>vaapi</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>vdpau</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>none</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>other</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="hwdecEdit">
       <property name="maximumSize">
        <size>
         <width>200</width>
         <height>16777215</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="3" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QPushButton" name="okBtn">
       <property name="text">
        <string>Confirm</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cancelBtn">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
